Aptoide/bds-sdk
# [APPC-1624] Pay as guest - Translations

**What does this PR do?**

   Refactors all of translation logic
   Adds translations to code.

**Database changed?**

  No

**Where should the reviewer start?**

TranslationRepository.java

**How should this be manually tested?**

Change language to PT and check if there's any english string or misplaced string

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1624

**Code Review Checklist**
